### DISCUSSION TITLE
2024 Q1 - Quarterly Brigade Planning Discussion

### DISCUSSION THREAD

**Discussion by tyliec** [CodeWithAloha/docs#5] [Q&A]
# Introduction

Greetings, Traveler 🧇 . As the time grows near upon our [upcoming quarterly brigade planning meeting](https://www.meetup.com/code-with-aloha/events/298029514/), I'd like to take this opportunity to open the floor to any potential feedback/topics to cover/suggested improvements. So far when we've had these planning meetings we might have had _some_ structure, but for the most part participants were coming in somewhat blind. As mentioned in the [**Brigade Unwrapped: A memorandum of 2023**](https://github.com/orgs/CodeWithAloha/discussions/3), one of the improvements aimed at this year is improving the way we make decisions. To that end, this discussion post is meant to serve as an early platform of things to talk about, and is open for anyone to contribute/suggest any comments/concerns/improvements. 🚀 

# Proposed Topics

Here are some proposed topics for the upcoming quarterly brigade planning meeting:

### Onboarding Review

As for onboarding, the state of the world consists of someone giving their variation of an onboarding walkthrough and the existing [onboarding guide](https://github.com/CodeWithAloha/docs/blob/01ecbb16fe07fee26c8078e1f87b706b6eb92bcf/ONBOARDING.md).

As part of improving the onboarding process, a new onboarding discussion thread: #4 has been created as a way to ease people into an early "contribution" & invitation to the group - as well as putting some of the Meetup group initial join question responses into a somewhat parseable format.

@kobebuckley created two repos that might be used in conjunction/replace the existing onboarding guide here:
- https://github.com/CodeWithAloha/onboardingGuide
- https://github.com/CodeWithAloha/gitBasics

### Taking on a new project

What could be more exciting than talking about taking on a new project? @Alexandra-Haynes has previously reached out to Jennifer Lynch from the Center for Marine Debris Research after the group's participation in HACC2023, however we haven't gotten a response. 

### Current Projects

Surveying the existing landscape, our current projects consist of:

- [Hawaii Zoning Atlas](https://hawaiizoningatlas.com/)
- [HIERR](https://github.com/CodeWithAloha/HIERR)
- [UIPA](https://github.com/CodeWithAloha/uipa)

We should talk about how each of these are going, and any existing work items that need to be done.

### Field Trips and Eligibility

Upcoming field trip: https://www.meetup.com/code-with-aloha/events/298229547/

![toothless-dragon-toothless](https://github.com/CodeWithAloha/docs/assets/15609358/1bae0121-e624-4e96-8ef3-0338a48d8ec6)

# Related Items

- Meetup Link to the upcoming Quarterly Brigade Planning Meeting: https://www.meetup.com/code-with-aloha/events/298029514/
- Previous Discussion Post: [**Brigade Unwrapped: A memorandum of 2023**](https://github.com/orgs/CodeWithAloha/discussions/3)

**Comment by avenmia** [comment]
I'd add the website to the current projects. I think we should go over rough goals for each of the projects and follow up on what is happening with Habitat for Humanity. Looks good though, thanks for putting this together!

**Comment by MichaelTamaki** [comment]
I've noticed a common pattern of new developers coming in that don't have much web dev experience. My observation is that once they get released into the wild after onboarding, they get overwhelmed. 

One possible solution we could explore is creating a space for new devs to build up their skills (unrelated to projects). Like one of the "breakout rooms" could be "web dev class" etc. Maybe direct them to something like https://roadmap.sh/frontend and code with aloha meetings can act like "office hours" where they can get support.

One of the points our org publicly advertises is that this is a community for all experience levels. Making a more intentional effort to support the beginner level would help us meet that ideal and help build the community. Not sure if we have the resources to support that though... Just throwing out the idea.

**Comment by dinoboy197** [comment]
I think that spending some time within each project to do a retrospective of how the last year went would be useful. It might yield insights about things that went well that kept projects on-track, interesting to contributors, and successful. Similarly, it might indicate things to stop doing or avoid in the future. Having learnings like these could be really useful to project leads for both current and future projects.

To facilitate participation, a Google Sheet could be generated for each project and contributors could add info asynchronously on their own time (what went well, what didn't go well, what things should the project consider in the future, etc). Each project could take a couple of weeks to pull in contributions, and then whoever wants to chat about them (on a project-by-project basis) could meet virtually in real-time to discuss. Final results could then be presented in one of the general meetings or discussed by brigade planning.

**Reply by tyliec** [reply]
Noting your idea [here](https://hicapacity.slack.com/archives/C05SZ7LH3D2/p1702860649688009) as well, 

```
Hey folks! I stumbled across the [All Contributors](https://allcontributors.org/) open-source project contribution tracker today. I have no idea if this is what we're using for the website, but I thought I'd mention it in case anyone is interested in reviewing. It's goal is to ensure that everyone who contributes to CFA, even those who are not "coders", are recognized.
```
